### PR TITLE
Add  step to update operator operand managementstate

### PR DESCRIPTION
**Pull request by wewang58**
@openshift/devexp-qe pls help to review it, thx

### CODE REVIEW

File: features/step_definitions/operator.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Given /^I updated the operator crd "([^"]*)" managementstate operand to (Managed|Removed|Unmanaged)$/ do |cluster_operator, manage_type|
```
Comment (xiuwang) [inline features/step_definitions/operator.rb]: s/I/admin ?

File: features/step_definitions/operator.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  ensure_admin_tagged
+  ensure_destructive_tagged
+  step %Q/I run the :patch client command with:/, table(%{
```
Comment (xiuwang) [inline features/step_definitions/operator.rb]: Should not be 'admin command'?

Comment (wewang58) [reply features/step_definitions/operator.rb]: Yes, should admin, updated

**Comment by xiuwang** [comment]
LGTM

**Pull request merged**